EventsExpress.Db/Enums/ContactAdminReason.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace EventsExpress.Db.Enums
+{
+    public enum ContactAdminReason
```

Comment:
Please, make it short at least. We don't need to spend 4 bytes for a small column.